src/components/ProgressIndicator/_progress-indicator.scss

```diff
@@ -2,61 +2,82 @@
 @import '../../globals/vars';
 
 .#{$iot-prefix}--progress-indicator {
-  .#{$prefix}--skeleton .#{$prefix}--progress-step {
-    flex: unset;
-  }
-
   &.#{$prefix}--progress--vertical {
-    .#{$prefix}--progress-label {
-      text-align: left;
-    }
-
     .#{$prefix}--progress-step-button {
+      display: list-item;
       padding: 0 $spacing-03;
       text-align: left;
       width: 100%;
+      min-height: unset;
+    }
 
-      .#{$prefix}--progress-step--current svg,
-      svg {
-        margin-left: 0;
-        width: 1rem;
-      }
+    .#{$prefix}--progress-step svg {
+      margin-left: 0.563rem;
```

Comment:
You could perhaps put this in a sass var and name it since it is used in more than one places?